### PR TITLE
update Nuxt Pages guide

### PR DESCRIPTION
**Pull request by dario-piotrowicz**

### CODE REVIEW

File: content/pages/framework-guides/nuxt.md [hunk 1]
```diff
@@ -81,14 +74,12 @@ Additionally, you will have access to [preview deployments](/pages/platform/prev
 For the complete guide to deploying your first site to Cloudflare Pages, refer to the [Get started guide](/pages/get-started/).
 
 {{</Aside>}}
-  
-{{<Aside type="warning" header="Auto Minify and Nuxt.js">}}
 
-If you are going to use a custom domain that is proxied through Cloudflare, you may run into issues with the Auto Minify feature. If enabled for HTML, it will strip comments which Nuxt.js and some other frameworks may rely on.
```
Comment (dario-piotrowicz) [inline content/pages/framework-guides/nuxt.md:87]: I've tried to set a custom domain for a Nuxt app and it looks like it's working fine (https://my-nuxt-app.dariopiot.net/) so I think this info may be outdated

File: content/pages/framework-guides/nuxt.md [hunk 2]
```diff
@@ -3,26 +3,25 @@ pcx_content_type: how-to
 title: Deploy a Nuxt.js site
 ---
 
-# Deploy a Nuxt.js site
+# Deploy a Nuxt site
 
-[Nuxt.js](https://nuxtjs.org/) is a framework making web development simple and powerful.
+[Nuxt](https://nuxt.com) is a web framework making web Vue.js based development simple and powerful.
```
Comment (deadlypants1973) [inline content/pages/framework-guides/nuxt.md]:
```suggestion
[Nuxt](https://nuxt.com) is a web framework making web Vue.js-based development simple and powerful.
```

File: content/pages/framework-guides/nuxt.md [hunk 3]
```diff
@@ -3,26 +3,25 @@ pcx_content_type: how-to
...
-$ npx create-nuxt-app my-nuxt-app
+$ npx nuxi init my-nuxt-app
+$ cd my-nuxt-app
+$ npm install
 ```
 
-You will be prompted to fill in various fields, such as `Project name`, `UI option`, `Linting tools`, and `Testing framework`, in your terminal. When asked which deployment target to use, you will be given two options: **Server** and **Static**. Select **Static** as your option for hosting.
-
-Next, navigate to the project and run it using the command:
+Next, you can run the application using the command:
```
Comment (deadlypants1973) [inline content/pages/framework-guides/nuxt.md]:
```suggestion
Next, run the application using the command:
```

File: content/pages/framework-guides/nuxt.md [hunk 4]
```diff
@@ -81,14 +74,12 @@ Additionally, you will have access to [preview deployments](/pages/platform/prev
 For the complete guide to deploying your first site to Cloudflare Pages, refer to the [Get started guide](/pages/get-started/).
 
 {{</Aside>}}
-  
-{{<Aside type="warning" header="Auto Minify and Nuxt.js">}}
 
-If you are going to use a custom domain that is proxied through Cloudflare, you may run into issues with the Auto Minify feature. If enabled for HTML, it will strip comments which Nuxt.js and some other frameworks may rely on.
+## Using bindings in your Nuxt application
```
Comment (deadlypants1973) [inline content/pages/framework-guides/nuxt.md]:
```suggestion
## Use bindings in your Nuxt application
```

File: content/pages/framework-guides/nuxt.md [hunk 5]
```diff
@@ -81,14 +74,12 @@ Additionally, you will have access to [preview deployments](/pages/platform/prev
...
 
-Refer to [Using Cloudflare Auto Minify](https://support.cloudflare.com/hc/en-us/articles/200168196-Using-Cloudflare-Auto-Minify) for how to configure this feature.
+Currently it is not possible to access Bindings from a Nuxt application (see this [Nuxt GitHub issue](https://github.com/nuxt/nuxt/issues/18599)).
```
Comment (deadlypants1973) [inline content/pages/framework-guides/nuxt.md]:
```suggestion
It is not currently possible to access bindings from a Nuxt application (refer to this [Nuxt GitHub issue](https://github.com/nuxt/nuxt/issues/18599)).
```

File: content/pages/framework-guides/nuxt.md [hunk 6]
```diff
@@ -81,14 +74,12 @@ Additionally, you will have access to [preview deployments](/pages/platform/prev
...
 
-{{</Aside>}}
+As soon as the Nuxt issue is addressed and Bindings are accessible we'll update this guide accordingly.
```
Comment (deadlypants1973) [inline content/pages/framework-guides/nuxt.md]:
```suggestion
When bindings are accessible from a Nuxt application, we will update this guide accordingly.
```

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**